docs/userGuide/reusingContents.md

```diff
@@ -259,6 +269,32 @@ In other words, **`<include>` interprets the reused code relative to the origina
 
 <hr><!-- ======================================================================================================= -->
 
+##### Specifying Variables in an `<include>`
+
+**It is possible to include variables in an `<include>`**
+
+<div class="indented">
+
+{{ icon_example }} Specifying `title` and `author` variables in an `<include>` tag:
+
+```html
+<include src="article.md">
+  <span id="title">My Title</span>
+  <span id="author">John Doe</span>
+</include>
+```
+
+In `article.md`:
+
+<code>
+# {<span></span>{ title }}
+
+Author: {<span></span>{ author }}
+</code>
+</div>
+
+These variables work the same way as variables in `_markbind/variables.md`, except that they only apply to the included file. They allow the included file to be reused as a template, for different source files using different variable values.
+
```

Comment:
The documentation can explain the precedence of variables (global > outer include > inner include).